contracts/implementations/MultipleMinter.sol

```diff
@@ -0,0 +1,140 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.13;
+
+import "./nodes/VehicleInternal.sol";
+import "./nodes/SyntheticDeviceInternal.sol";
+import "../interfaces/INFT.sol";
+import "../Eip712/Eip712CheckerInternal.sol";
+import "../libraries/NodesStorage.sol";
+import "../libraries/nodes/ManufacturerStorage.sol";
+import "../libraries/nodes/IntegrationStorage.sol";
+import "../libraries/nodes/VehicleStorage.sol";
+import "../libraries/nodes/SyntheticDeviceStorage.sol";
+import "../libraries/MapperStorage.sol";
+
+import "../shared/Roles.sol";
+
+import "@solidstate/contracts/access/access_control/AccessControlInternal.sol";
+
+error DeviceAlreadyRegistered(address addr);
+error InvalidSdSignature();
+
+contract MultipleMinter is
+    AccessControlInternal,
+    VehicleInternal,
+    SyntheticDeviceInternal
+{
+    bytes32 private constant MINT_VEHICLE_SD_TYPEHASH =
+        keccak256("MintVehicleAndSdSign(uint256 integrationNode)");
```

Comment:
We have
```
MintVehicleSign(uint256 manufacturerNode,address owner,string[] attributes,string[] infos)
```
does this seem a little thin, by comparison?

Reply:
Ohh, the owner signs `MintVehicleSign` still? This is difficult to follow.

Reply:
Yep, afterall the main thing here is still the vehicle minting I suppose.